ci-operator/step-registry/configure-registry-storage/destroy-s3-cloudfront/configure-registry-storage-destroy-s3-cloudfront-commands.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -f "${SHARED_DIR}/create_aws_bucket.tf" ]]; then
```

Comment:
I think we can make the file name more clear, e.g `create_s3_bucket_for_registry_storage`